### DISCUSSION TITLE
How may donations be received, stored and distributed?

### DISCUSSION THREAD

**Discussion by wilsonowilson** [DevsHelpDevs/DevsHelpDevs#7] [Q&A]
In other words, what _international_ payment and payout service would be used in such a site?

**Comment by escamoteur** [comment]
First of all, I don't want to handle any money on this platform if we can avoid it. Which means we need partners for that.
For many countries the choice to transfer money is very limited, so that Western Union will be probably one such partner. Paypal Xoom could be one too if they were willing to change their policy for us.

1. Ideally money would flow directly from donor to either applicants or to a trusted shop that will ship the hardware. The platform will only act as Guide to make it as easy as possible.
2. For money pools like Gofundme we need a partner who can do that independent where the receiver will be. Maybe Gofundme could do that. https://flutterwave.com/us/ could possibly also such a partner. This would be for individual pools per applicant.
3. We probably also want to be able to offer a regularly donation to a platform wide pool that could either be distributed automatically to the individual pools or me managed by a platform comitee deciding who gets supported by this. 

I haven't reached out to any of these platforms yet.

**Reply by dev-darrell** [reply]
I totally agree with the points you made here. It would be best if we didn't handle money directly.

**Reply by 805702** [reply]
I'm totally okay with the idea of having partnered shops for hardware. this will avoid funny issues and will permit money to be sent in, in bids and escape the idea of crowdfunding whereby the money is sent in bids and when it is complete before it is sent to the applicant in question.

also, it assures us that there is no fraud given that the money goes to a local store who will later on ship the machine/machine part to the applicant in question. I think it's actually very good as an idea.

**Reply by escamoteur** [reply]
>l permit money to be sent in, in bids and escape the idea of crowdfunding
I think I didn't understand what you want to say by that.

**Reply by 805702** [reply]
crowdfunding will require the platform to either have some party that will keep this money but with the idea of partnered shops, we can actually have the shops register unto the platform and the money from the donors is sent directly to these shops and when the money is enough for the part, it is shipped to the applicant directly.

fault: what happens if the money is not never enough but in the mean time, some money has already been sent to the local shop?

**Reply by escamoteur** [reply]
Yep, exactly this latest part is a problem I have no idea yet how to solve. But it also occurs with a crowd funding unless you set a time limit and refund the money/never really charge depending on the system.

**Reply by escamoteur** [reply]
Maybe the system could favor listing applications that have already received some significant funds on the starting page. It doesn't help anybody if we have 100 donations but noone gets something because they are split on 100 applications and don't reach their goal

**Reply by 805702** [reply]
So i think we rather adopt that as the solution and let's look now at ways to solve the fault. 
I'll give it a deeper thought during my free time and I'll certainly bring up better ideas to solve that up.

**Comment by biggaji** [comment]
For the payments, I think we can use stripe or paystack.
They both support international transactions.

**Reply by escamoteur** [reply]
But they only offer business solutions where we would have to handle the money but we can check them.

**Reply by biggaji** [reply]
Yeah, that's true!

**Reply by StephenLadoye** [reply]
Paystack offer business solution and support NGos too. There's a limit to what you can receive if you're not a registered organization. Moreover, they will remove their cut from each transaction. For instance, they will remove 15 naira from every 1000 naira received on the platform

**Comment by Jhanbury** [comment]
I think Item 3 may force the site to have some responsibility for donations.
Unless we can do this via a providers REST API? 
We would have to use something like Stripe for this process as @biggaji illuded to.
I'm not entirely familiar with this form of the payment flow (dividing a donation to multiple parties) so it may need some more investigation to get a sense of what options we have?

**Reply by escamoteur** [reply]
Point 3 I see probably in v2+ because as you mentioned we would need some external partner for this. maybe gofundme or opencollective could help here.
And I really want to avoid haveing the platform taking over finacnial resposibility

**Reply by Jhanbury** [reply]
I agree with your point on not wanting to handle any financial processing

**Comment by escamoteur** [comment]
I would like to pick up this discussion again. I think there are different ways we can make this work

* Find trustful local dealers that can execute repairs or sell parts when being paid directly by the donor
* Same for used laptops
* find a way to collect money like gofundme so that the applicant only gets the money when the full amount is reached

I need your expertise on your countries. Is it realistic to find such partners in your country? In the end they would get regular revenue for that that they might have a bit more work?

## collect money
From your comments in the requirements doc I take that offering such a model seems important for many of us. for this we will need a solution that:
* Allows to manage individual accounts for each applicant
* Handles all money related things
* If possible only transfers the money to a trustworthy dealer who will then ship the laptop/part

(actually if we do that we should add a filter to the homepage to show already started collections

### potential partners

#### Flutterwave
From what I found so far https://flutterwave.com/us/  could be such a partner. Possibly their https://flutterwave.com/us/card-issuing could be used to generate separate accounts but we should directly discuss that with them.
From reviews it seems that their customer supports isn't great. 
Does anyone of you have experience with them?

##### Ulule
https://ulule.com/
could make sense to get in touch with them

Unfortunately I didn't find any other options so far.

Could it be an option to have a local dealer collecting the money for donations? But what if they don't get completed? And would someone do this? For Africa Flutterwave seems like the most realistic because they support payments in a number of countries. I tried to reach out to two customers of them to hear what their experience with them is, but didn't get a response yet.


## Money isn't everything
Besides money transfer we should encourage especially people from the US to ship their old laptops as it seems quire easy with https://heroshe.com/ which could be a partner.  For other countries than Nigeria we have to find other shipping companies


Please add your thoughts.
If you know really reliable dealers please also list them here.

**Reply by mugisha23** [reply]
I think the main problem with dealers that can execute repairs or sell parts and you can trust, is that they are usually expensive.
If possible to the donor could just ship computer or send money to buy or repairs laptop

**Reply by escamoteur** [reply]
Maybe we could offer both as options where the one over the dealer is more expensive? Would it make sense to negotiate prices with this dealers for certain repairs or parts? They will get business through us that they wouldn't get otherwise.

So please if you know possible dealers, lets collect them and try to talk with them.

**Reply by mugisha23** [reply]
In Burundi(where i'm located) there's probably few trustful dealers i can recommend: [maison zarine](http://www.maisonzarinebi.com/products.html) and [zebra electronics](https://www.zebra-electronics.org/).  you can usually get what they sell for a cheaper price in other small stores or just people but that would requires me to handle money directly

**Reply by StephenLadoye** [reply]
Yes, we can send and receive international payments via flutterwave.  All we need to do is to use flutterwave APIs

**Reply by 805702** [reply]
Expensive to me is not a problem the problem is being sure that the transaction that was started runs to the end. 
for this to be done successfully, we need a group of people whom we can trust (registered structures) who will submit their registration documents (registre de commerce and taxation card for Cameroun) to show that they are in line with the government. 

they have their own portal on the platform that permits them to see the progress of things having to deal with them. 
in this way, if anything goes wrong with them, we know who is entitled to that and we can easily let them settle that with the law... I think the local dealers and repairers are the best short. For Africa in particular.

**Comment by seunbayo** [comment]
For transparency issue and effective monitoring,there is no way  (helpDevs) wont be keeping money because

1. We need to first identify partners online that can sell laptops,ship laptops,repair laptops and i mean dealers with verified business online.

2. People wont like the idea of just donating directly to a dealer or individual because there will be no way to audit those accounts at the end of the day.
3. We should seek more fintech partners and see if they could reduce charges on every donations made and at the end of a quarter make/publish  financial accounts for transparency sake
